data/migrations/versions/e9f3e4dbb979_quota_management_reporting.py

```diff
@@ -0,0 +1,88 @@
+"""quota_management_reporting
```

Comment:
We should split the migration into a separate PR. That way, we can perform the migration first, and then if there's an issue when deploying, we can revert the image without having to rollback on the migration itself.

Reply:
This makes sense I will get it going